tests/integ/test_packaging.py

```diff
@@ -408,13 +413,34 @@ def test_add_unsupported_requirements_twice_should_not_fail_for_same_requirement
     IS_IN_STORED_PROC,
     reason="Subprocess calls are not allowed within stored procedures.",
 )
+@pytest.mark.skipif(
+    not runtime_39_or_above, reason="arch is not available on python 3.8"
```

Comment:
Just curious, why do we have to use `arch` for testing? Does it bring in a different version of `scipy` or `pandas` that satisfies our test scenario?

Reply:
this is the scenario that failed in the issue raised by users so I'm using the same example. it downloads `scipy` and `pandas` that triggers the scenario mentioned above.